polarization/tissueLayer.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, birefringence, opticAxis, scattDensity=0, thickness=200):
         self.scattDensity = scattDensity
         self.thickness = thickness
 
+        self._K = []
```

Comment:
Simpler solution: a dictionary with the key being the wavevector
`self.matrixCache = {k1:JonesMatrix,k2:JonesMatrix, etc...}`
see below for usage.